homeassistant/components/image_processing/__init__.py

```diff
@@ -119,20 +120,49 @@ async def async_scan_service(service: ServiceCall) -> None:
     return True
 
 
+@dataclass
+class ImageProcessingEntityDescription(EntityDescription):
+    """A class that describes sensor entities."""
+
+    device_class: ImageProcessingDeviceClass | None = None
+    camera_entity: str | None = None
+    confidence: float | None = None
+
+
 class ImageProcessingEntity(Entity):
     """Base entity class for image processing."""
 
-    _attr_device_class: ImageProcessingDeviceClass | str | None
+    entity_description: ImageProcessingEntityDescription
+    _attr_device_class: ImageProcessingDeviceClass | None
+    _attr_camera_entity: str | None
+    _attr_confidence: float | None
     timeout = DEFAULT_TIMEOUT
 
     @property
     def camera_entity(self) -> str | None:
         """Return camera entity id from process pictures."""
+        if hasattr(self, "_attr_camera_entity"):
+            return self._attr_camera_entity
+        if hasattr(self, "entity_description"):
+            return self.entity_description.camera_entity
         return None
 
     @property
     def confidence(self) -> float | None:
         """Return minimum confidence for do some things."""
```

Comment:
Should the docstring be adjusted?
```suggestion
        """Return minimum confidence to do some things."""
```